src/test/java/fr/inria/coming/spoon/repairability/repairtools/JGenProgTest.java

```diff
@@ -1,24 +0,0 @@
-package fr.inria.coming.spoon.repairability.repairtools;
-
-import fr.inria.coming.changeminer.entity.FinalResult;
-import fr.inria.coming.spoon.repairability.TestUtills;
-import org.junit.Test;
-
-/**
- * TODO: Add more than one class in a file test: patch4-Chart-15-JGenProg2017
- */
-public class JGenProgTest {
```

Comment:
don't remove the JGenProgTest we need one test file per repair system

Reply:
but the file is present in the master, as I understand I should not have it in the  Arja PR, because it should not be changed in Arja PR.